### PR TITLE
V11 1

### PR DESCRIPTION
**Pull request by Yuri05**
PR can be merged after 11.1 docu is finalized (and before the final release of 11.1)

### CODE REVIEW

File: part-5/parameter-identification.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ You can select between three optimization algorithms, edit their standard settin
 
 ### Run Optimization‌
 
-After finishing the previous steps, you can run a Parameter Identification which does not block the application. So you can proceed with manual work in your project. However the changes made in referenced simulations are not reflected in the _running_ Parameter Identification.
+After finishing the previous steps, you can run a Parameter Identification through the “Run & Analyze” tab. Running the Parameter Identification does not block the application, so you can proceed with manual work in your project. However the changes made in referenced simulations are not reflected in the _running_ Parameter Identification. Multiple Parameter Identifications can be run in parallel. The “Run & Analyze” tab reflects the state of the currently selected Parameter Identification.
```
Comment (Yuri05) [inline part-5/parameter-identification.md:59]: _"The “Run & Analyze” **tab** reflects the state of the currently selected Parameter Identification."_

Let's replace **tab** with **Ribbon Bar** 
(compare e.g. https://docs.open-systems-pharmacology.org/shared-tools-and-example-workflows/parameter-identification#simple-example
grafik

**Comment by Yuri05** [comment]
@georgeDaskalakis 
assets/images/part-3/TwoFoldDeviationLine.png still has the old "Data" tab...

File: part-5/parameter-identification.md (hunk 1 above)
Comment (georgeDaskalakis) [reply part-5/parameter-identification.md]: done

**Comment by Yuri05** [comment]
As for me, this can be merged now.
Any objections?

**Comment by georgeDaskalakis** [comment]
From my side I also thiknk the docu is finalized.

**Review by PavelBal — APPROVED** [review]

**Review by rwmcintosh — APPROVED** [review]

**Pull request merged**